packages/@webex/plugin-meetings/src/locus-info/index.ts

```diff
@@ -93,18 +93,25 @@ export default class LocusInfo extends EventsScope {
    * Does a Locus sync. It tries to get the latest delta DTO or if it can't, it falls back to getting the full Locus DTO.
    *
    * @param {Meeting} meeting
+   * @param {boolean} isLocusUrlChanged
    * @returns {undefined}
    */
-  private doLocusSync(meeting: any) {
+  private doLocusSync(meeting: any, isLocusUrlChanged: boolean) {
     let isDelta;
     let url;
     let meetingDestroyed = false;
 
-    if (this.locusParser.workingCopy.syncUrl) {
+    if (this.locusParser.workingCopy?.syncUrl) {
       url = this.locusParser.workingCopy.syncUrl;
       isDelta = true;
     } else {
-      url = meeting.locusUrl;
+      if (isLocusUrlChanged) {
```

Comment:
shouldn't this check be done at the beginning, before the check for `(this.locusParser.workingCopy?.syncUrl)`?